src/runtime/volatile/register_nxpmk66f18.go

```diff
@@ -0,0 +1,79 @@
+// +build nxp,mk66f18
+
+package volatile
+
+import "unsafe"
+
+const registerBase = 0x40000000
+const registerEnd = 0x40100000
+const bitbandBase = 0x42000000
```

Comment:
Your comments on #842 mentioned wanting to limit the number of places developers are required to make changes. Defining these in `runtime/volatile` would add a place. Of course, if `bitband.go` used the build tags `cortexm,bitband`, then targets could opt in. I don't think the constants could be defined in `machine`, `runtime`, or even `device/*` because it would cause a circular reference.